api/src/paths/project/{projectId}/survey/{surveyId}/summary/submission/upload.ts

```diff
@@ -207,12 +169,13 @@ export function uploadMedia(): RequestHandler {
         email: (req['auth_payload'] && req['auth_payload'].email) || ''
       };
 
+      // Upload submission to S3
       await uploadFileToS3(rawMediaFile, key, metadata);
 
-      req['s3File'] = rawMediaFile;
+      // Validate submission
+      await summaryService.validateFile(summarySubmissionId, surveyId);
 
-      req['summarySubmissionId'] = summarySubmissionId;
-      next();
+      return res.status(200).json({ summarySubmissionId });
     } catch (error) {
       defaultLog.error({ label: 'uploadMedia', message: 'error', error });
       await connection.rollback();
```

Comment:
I think we want to insert a submission status here indicating a system error. You can look at `api/paths/xlsx/validate` for an example.